pkg/devfile/parser/context/fake.go

```diff
@@ -0,0 +1,10 @@
+package parser
```

Comment:
all fields in`DevfileCtx` are defined as internal, and cannot be accessible outside of the context package

Reply:
You are right.  Let's move FakeContext() to xxx_test.go so that it is not visible outside of tests.   Also, we should consider not exporting this function.

Reply:
I agree we should rename the file to avoid confusion. 
The function should be kept as exported, because consumers(like odo) would need that function for their testing purpose.